src/features/query-bar/components/filter/FilterButton.js

```diff
@@ -148,18 +164,74 @@ class FilterButton extends React.Component<Props, State> {
         }
     };
 
-    deleteCondition = (index: number) => {
-        const { conditions } = this.state;
+    updateSelectedPrefix = (option: Object) => {
+        const displayText = option.displayText;
+        let updatedPrefix = '';
+
+        switch (displayText) {
+            case 'AND':
```

Comment:
the string 'AND' and 'OR' are available as constants as you used them below

Reply:
Fixed